packages/ethers/src.ts/index.ts

```diff
@@ -59,8 +59,6 @@ export {
     PayableOverrides,
     CallOverrides,
 
-    PopulatedTransaction,
```

Comment:
I think that we should be exporting this?

Reply:
yep, reverted